### PR TITLE
Subscription Method Added

### PR DESCRIPTION
**Pull request by inf3cti0n95**
Handling subscriptions with WebSocket Multiplexing in RxJS

*message filtering to be written be written properly. remainning because of lack of documentation*

### CODE REVIEW

File: src/RealTimeAPI.ts [hunk 1]
```diff
@@ -189,4 +189,28 @@ export class RealTimeAPI {
...
+                "params": [
+                    streamParam,
+                    addEvent
+                ]
+            }),
+            () => JSON.stringify({
+                "msg": "unsub",
+                "id": id
+            }),
+            (message: any) => typeof message.collection === "string" && message.collection === streamName && message.fields.eventName === streamParam // Proper Filtering to be done. This is temporary filter just for the stream-room-messages subscription
```
Comment (inf3cti0n95) [inline src/RealTimeAPI.ts]: Message Filtering remaining, This should filter messages for the subscription.

Comment (inf3cti0n95) [inline src/RealTimeAPI.ts]: There is lack of documentation in realtime api, I don't know the response messages for the subscriptions.

File: lib/RealTimeAPI.js [hunk 2]
```diff
@@ -169,6 +169,26 @@ var RealTimeAPI = (function () {
...
+            "id": id,
+            "name": streamName,
+            "params": [
+                streamParam,
+                addEvent
+            ]
+        }); }, function () { return JSON.stringify({
+            "msg": "unsub",
+            "id": id
+        }); }, function (message) { return typeof message.collection === "string" && message.collection === streamName && message.fields.eventName === streamParam; } // Proper Filtering to be done. This is temporary filter just for the stream-room-messages subscription
```
Comment (ggazzo) [inline lib/RealTimeAPI.js]: why cant you filter by id here?

**Comment by inf3cti0n95** [comment]
This method of filtering seems fine for now and thus merging the PR ;)

**Pull request merged**